### PR TITLE
Minor code simplification

### PR DESCRIPTION
**Pull request by PiergiorgioButtarini**
Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibolab/instruments/qblox/cluster_qcm_bb.py [hunk 1]
```diff
@@ -287,7 +282,6 @@ def _get_next_sequencer(self, port, frequency, qubits: dict):
             # TODO: Throw error in that event or implement for non_overlapping_same_frequency_pulses
             # Even better, set the frequency before each pulse is played (would work with hardware modulation only)
 
-            self.ports[port].offset = qubit.sweetspot if qubit else 0
```
Comment (PiergiorgioButtarini) [inline src/qibolab/instruments/qblox/cluster_qcm_bb.py:290]: This is no longer needed since the sweetspot is assigned during the platform setup().

**Review by stavros11 — APPROVED** [review]

File: src/qibolab/instruments/qblox/cluster_qcm_bb.py [hunk 2]
```diff
@@ -252,17 +252,12 @@ def _get_next_sequencer(self, port, frequency, qubits: dict):
...
         for _qubit in qubits.values():
-            if _qubit.flux.port is not None:
-                if (
-                    _qubit.flux.port.name == port
-                    and _qubit.flux.port.module.name == self.name
-                ):
-                    qubit = _qubit
-            else:
-                log.warning(f"Qubit {_qubit.name} has no flux line connected")
+            if _qubit.flux and _qubit.flux.port == self.ports[port]:
```
Comment (AleCandido) [inline src/qibolab/instruments/qblox/cluster_qcm_bb.py:258]:
```suggestion
            if _qubit.flux is not None and _qubit.flux.port == self.ports[port]:
```
Don't use the bare truthiness of the attribute, it is error-prone.

**Review by AleCandido — APPROVED** [review]
Fine, but it would be better if you could apply the suggestion before merging (either with the `is not None` or another explicit comparison, whatever is needed)

File: src/qibolab/instruments/qblox/cluster_qcm_bb.py (hunk 2 above)
Comment (PiergiorgioButtarini) [reply src/qibolab/instruments/qblox/cluster_qcm_bb.py:258]: Thanks!

**Pull request merged**